web/files/paginated.go

```diff
@@ -354,12 +354,19 @@ func (f *file) Links() *jsonapi.LinksList {
 		}
 	}
 	if f.doc.Class == "pdf" {
+		log := f.instance.Logger().WithNamespace("files")
+		start := time.Now()
+
 		if secret, err := vfs.GetStore().AddIcon(f.instance, f.doc.DocID); err == nil {
 			links.Icon = "/files/" + f.doc.DocID + "/icon/" + secret
 		}
 		if secret, err := vfs.GetStore().AddPreview(f.instance, f.doc.DocID); err == nil {
 			links.Preview = "/files/" + f.doc.DocID + "/preview/" + secret
 		}
+		elapsed := time.Since(start)
+		if log.IsDebug() {
+			log.Infof("Time elapsed to get PDF links: %v\n", elapsed)
```

Comment:
Why not `log.Debugf`?

Reply:
Because I forgot its existence :) With `log.Debugf`, no need for `log.IsDebug()`, I assume?

Reply:
yes, you can remove the `IsDebug`

Reply:
Done